product_pack/i18n/es.po

```diff
@@ -0,0 +1,422 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * product_pack
+# 
+# Translators:
+# Juan José Scarafía <scarafia.juanjose@gmail.com>, 2019
+# Gonzalo Ruzafa <gr@ingadhoc.com>, 2019
+# emiquinodoz <eq@ingadhoc.com>, 2019
+# Julián García <jg@adhoc.com.ar>, 2019
+# Valentino Defelice <vd@adhoc.com.ar>, 2019
+# Santiago Crespo <sc@adhoc.com.ar>, 2019
+# Mariana Luz Torres <mt@adhoc.com.ar>, 2019
+# Nicolas Mac Rouillon <nmr@ingadhoc.com>, 2019
+# Katherine Zaoral <katherine.zaoral@gmail.com>, 2019
+# 
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-05-17 09:13+0000\n"
+"PO-Revision-Date: 2018-05-22 01:28+0000\n"
+"Last-Translator: Katherine Zaoral <katherine.zaoral@gmail.com>, 2019\n"
+"Language-Team: Spanish (https://www.transifex.com/adhoc/teams/46451/es/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Language: es\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: product_pack
+#: model:ir.model.fields,help:product_pack.field_product_product_pack_type
+#: model:ir.model.fields,help:product_pack.field_product_template_pack_type
+#: model:ir.model.fields,help:product_pack.field_sale_order_line_pack_type
```

Comment:
update the translation file, the sale_order* references to files / views / models does not exist anymore in this module